bank2ynab.conf

```diff
@@ -121,6 +121,14 @@ Source CSV Delimiter = ;
 Input Columns = skip,skip,Date,skip,Memo,Payee,skip,skip,Inflow,skip,skip
 Date Format = %%d.%%m.%%y
 
+[DE Ostseesparkasse Rostock credit card]
+Use Regex for Filename = True
+Source Filename Pattern = umsatz-[0-9]{4}________[0-9]{4}-[0-9]{8}
```

Comment:
I'd prefer that 8-digit dates have the "20" explicitly, in order to prevent mismatches on 8-digit non-date numbers. So I'd make this end in `20[0-9]{6}` instead. Agree?